### PR TITLE
update module imports

### PR DESCRIPTION
**Pull request by itdependsnetworks**
This partially addresses: https://github.com/napalm-automation/napalm/issues/575 and https://github.com/napalm-automation-community/napalm-panos/issues/50

On the communituy driver you still get: 

```
>>> driver = get_network_driver('panos')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "napalm/base/__init__.py", line 107, in get_network_driver
    .format(install_name=module_install_name))
napalm.base.exceptions.ModuleImportError: No class inheriting "napalm.base.base.NetworkDriver" found in "napalm.panos".
```

> napalm.base.base.NetworkDriver

You still have to do similar on community driver, but wanted to run by you all before adding a bunch of PR's. 

```
try:
    from napalm_base import NetworkDriver
except ImportError:
    from napalm_base.base import NetworkDriver
```

### CODE REVIEW

File: napalm/base/__init__.py [hunk 1]
```diff
@@ -79,12 +79,17 @@ def get_network_driver(module_name, prepend=True):
...
             module_install_name = 'napalm.{name}'.format(name=module_install_name)
+        # Order is custom_napalm_os (local only) ->  napalm.os (core) ->  napalm_os (community)
         try:
             module = importlib.import_module("custom_" + module_install_name)
         except ImportError:
-            module = importlib.import_module(module_install_name)
+            try:
+                module = importlib.import_module(module_install_name)
+            except ImportError:
+                module = importlib.import_module(community_module_name)
```
Comment (ktbyers) [inline napalm/base/__init__.py]: Can we just do a for loop?

custom_install_name = "custom_" + module_install_name
for module_name in [custom_install_name, module_install_name, community_module_name]:
    try:
        ...
        break
    except ImportError:
        pass
    else:
        # Never loaded
        raise ImportError(msg)
```

I think would be clearer to understand than the nested try

**Comment by dbarrosop** [comment]
@mirceaulinic weren't you working on this? Just making sure we don't conflict with each other.

Other than that LGTM

**Comment by mirceaulinic** [comment]
Yes, I did some work, but I didn't test it extensively yet. But I'm happy for @itdependsnetworks to go ahead with this one, no worries.

File: napalm/base/__init__.py [hunk 2]
```diff
@@ -73,24 +73,28 @@ def get_network_driver(module_name, prepend=True):
...
+    # Order is custom_napalm_os (local only) ->  napalm.os (core) ->  napalm_os (community)
+    for module_name in [custom_install_name, module_install_name, community_install_name]:
         try:
-            module = importlib.import_module("custom_" + module_install_name)
+            module = importlib.import_module(module_name)
+            break
         except ImportError:
-            module = importlib.import_module(module_install_name)
-    except ImportError:
+            pass
```
Comment (mirceaulinic) [inline napalm/base/__init__.py]: I'm not a big fan of doing this, but I guess that works okay.

Comment (itdependsnetworks) [reply napalm/base/__init__.py]: I did it without forloop, and @ktbyers suggested this way. I'm good either way.

Comment (dbarrosop) [reply napalm/base/__init__.py]: Not sure if there is a better way to do it but I certainly agree with @ktbyers this is nicer than the nested try and except we had before :)

**Pull request merged**